docs/widgets/media-server.mdx

```diff
@@ -0,0 +1,67 @@
+---
+title: 🎞️ Media Server
+tags:
+  - Widgets
+  - Integration
+  - Configuration
+  - Media Server
+  - Plex
+  - Jellyfin
+  - Sessions
+  - API
```

Comment:
```suggestion
  - Widgets
  - Integration
  - Configuration
```

Keep to the required tags.

Reply:
I disagree. Why would we remove those?
They help with indexing and the search. If you just enter Jellyfin, you'll find this page.

Reply:
They don't help with indexing and search. They are purely for categorizing on the tags page. Our search plugin can already search contents of the page.

Reply:
Okay, fair. I will apply your suggestion.